src/sql/expertise_outstanding_by_id.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
```

Comment:
@tkellen this query feels weird to me. I need to select all users that don't have an entry for this user + expertise combo in the log table, but where the expertise description is not null. Is there a more concise way to do this?